[2024-07-15T16:14:38.233+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-15T16:14:38.267+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Postgres_dag.insert_into_table manual__2024-07-15T16:14:36.136595+00:00 [queued]>
[2024-07-15T16:14:38.274+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Postgres_dag.insert_into_table manual__2024-07-15T16:14:36.136595+00:00 [queued]>
[2024-07-15T16:14:38.275+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-15T16:14:38.325+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-07-15 16:14:36.136595+00:00
[2024-07-15T16:14:38.331+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Postgres_dag', 'insert_into_table', 'manual__2024-07-15T16:14:36.136595+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmpu0zuk1g_']
[2024-07-15T16:14:38.334+0000] {standard_task_runner.py:91} INFO - Job 33: Subtask insert_into_table
[2024-07-15T16:14:38.335+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1255) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-15T16:14:38.336+0000] {standard_task_runner.py:63} INFO - Started process 1256 to run task
[2024-07-15T16:14:38.403+0000] {task_command.py:426} INFO - Running <TaskInstance: Postgres_dag.insert_into_table manual__2024-07-15T16:14:36.136595+00:00 [running]> on host 32f997017b5a
[2024-07-15T16:14:38.531+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rayzo' AIRFLOW_CTX_DAG_ID='Postgres_dag' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T16:14:36.136595+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T16:14:36.136595+00:00'
[2024-07-15T16:14:38.532+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-15T16:14:38.545+0000] {sql.py:276} INFO - Executing:  
        insert into drag_runs(dt,dag_id) values('2024-07-15', 'Postgres_dag')
        
[2024-07-15T16:14:38.554+0000] {base.py:84} INFO - Using connection ID 'Postgres_con' for task execution.
[2024-07-15T16:14:38.561+0000] {base.py:84} INFO - Using connection ID 'Postgres_con' for task execution.
[2024-07-15T16:14:38.574+0000] {sql.py:487} INFO - Running statement:  
        insert into drag_runs(dt,dag_id) values('2024-07-15', 'Postgres_dag')
        , parameters: None
[2024-07-15T16:14:38.579+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-15T16:14:38.613+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-15T16:14:38.626+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Postgres_dag, task_id=insert_into_table, run_id=manual__2024-07-15T16:14:36.136595+00:00, execution_date=20240715T161436, start_date=20240715T161438, end_date=20240715T161438
[2024-07-15T16:14:38.711+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-15T16:14:38.727+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-15T16:14:38.750+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
